pontoon/base/urls.py

```diff
@@ -30,6 +30,15 @@
     url(r'^contributor/(?P<email>[\w.%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4})/$',
         RedirectView.as_view(url="/contributors/%(email)s/", permanent=True)),
 
+    # List all active localization teams
+    url(r'^teams/$',
```

Comment:
Teams are something localizers can feel part of and they are already used to the expression:
https://wiki.mozilla.org/L10n:Teams
https://l10n.mozilla.org/teams/